src/test/mochitest/browser_dbg-outline.js

```diff
@@ -9,6 +9,20 @@ function getNthItem(dbg, index) {
   return findElement(dbg, "outlineItem", index);
 }
 
+// Tests that the outline functions for original and pretty printed source matches
```

Comment:
I'd probably clarify this, since this only verifies that they have the same number of items, not that they match.